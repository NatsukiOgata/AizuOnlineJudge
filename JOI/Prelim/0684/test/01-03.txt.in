30 20
2 2 26 25 2 61 65 2 8 26 25 61 61 61 25 61 83 17 83 26 81 81 17 83 34 34 26 8 81 8
8 61 17 61 25 2 81 65 8 26 81 26 25 65 2 25 81 2 81 17
