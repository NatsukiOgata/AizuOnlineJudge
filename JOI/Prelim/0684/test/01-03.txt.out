2
8
17
25
26
61
65
81
